src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
This was honestly the only extra thing that I felt needed an option. It's just jarring to go back and forth on a silly little thing. Sure, probably kind of petty in the long run, but it's almost like the tabs-vs-spaces argument to me. You're just never going to convince firm believers that one is better than the other.